support/lobby/lib/Review.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Table, Button } from "react-bootstrap";
+import dateFormat from "dateformat";
+
+export default function Reviews( props ) {
+  return <Table striped>
+    <thead>
+      <Head/>
+    </thead>
+    <tbody>
+      <Reviewws records={ props.records }/>
```

Comment:
Yes, this is some silliness I should have cleaned up. It was originally `Scenarios` since it's the set of `Scenario` elements, but I had to change it to make way for the containing `Scenarios` component. The component could have been unnamed, but the React debugger wouldn't identify the component in that case.